cypress/e2e/models/migration/reports/reports.ts

```diff
@@ -16,57 +16,64 @@ limitations under the License.
 /// <reference types="cypress" />
 /// <reference types="cypress-xpath" />
 
-import { applicationName, risk } from "../../../types/constants";
+import { SEC, applicationName, risk } from "../../../types/constants";
 import * as commonView from "../../../views/common.view";
 import { itemsPerPageMenu, itemsPerPageToggleButton } from "../../../views/reports.view";
 
 export function selectItemsPerPageAdoptionCandidate(items: number): void {
-    cy.get(itemsPerPageMenu)
-        .find(itemsPerPageToggleButton)
-        .then(($toggleBtn) => {
-            if (!$toggleBtn.eq(0).is(":disabled")) {
-                $toggleBtn.eq(0).trigger("click");
-                cy.get(commonView.itemsPerPageMenuOptions);
-                cy.get(`li > button[data-action="per-page-${items}"]`).click();
-                cy.wait(2000);
-            }
+    cy.log(`Select ${items} per page`);
+    cy.contains("h3", "Adoption candidate distribution")
+        .closest("div.pf-v5-l-stack__item")
```

Comment:
Please put selector to the corresponding 'view' file

Reply:
Due to the fact the the closest div that we are looking for as above and outside of the h3, i can't chain them in one selector, however i created a selector for the h3 title